modules/sdk-core/src/v2/environments.ts

```diff
@@ -172,7 +172,12 @@ export const Environments: Environments = {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     uri: process.env.BITGO_CUSTOM_ROOT_URI!,
     network: process.env.BITGO_CUSTOM_BITCOIN_NETWORK as V1Network,
-    hsmXpub: hardcodedPublicKeys.hsmXpub.dev,
+    hsmXpub:
+      process.env.NODE_ENV === 'production'
+        ? hardcodedPublicKeys.hsmXpub.prod
+        : process.env.NODE_ENV === 'test'
```

Comment:
This doesn't get to `test` set for testnet. We either need another way of detecting we're running in testnet mode or we need to add a new env var to represent the HSM key.

`NODE_ENV` is an env var used in many packages in the nodejs ecosystem to do things like hide debug logs, etc

Reply:
See https://nodejs.dev/learn/nodejs-the-difference-between-development-and-production for a bit more info on this.

Reply:
ahh I see this is the nodejs env